kratos/processes/simple_mortar_mapper_process.h

```diff
@@ -445,11 +407,11 @@ class SimpleMortarMapperProcess
      * @param ThisIntegrationMethod: The integration method considered
```

Comment:
you miss pointing to the thesis or work you refer to. page and formula as i commented you

Reply:
I added to the mortar_classes, here the mortar operators are used as black boxes